gateway/webhook/webhook.go

```diff
@@ -114,7 +115,7 @@ func (webhook *HandleT) RequestHandler(w http.ResponseWriter, r *http.Request) {
 	var postFrom url.Values
 	var multipartForm *multipart.Form
 
-	if r.Method == "GET" {
+	if r.Method == "GET" && !lo.Contains(webhook.config.forwardGetRequestForSrcs, sourceDefName) {
```

Comment:
We can probably use Map instead of slice here.
```
    // definition
    forwardGetRequestForSrcsMap map[string]struct{}

    // initialization
	webhook.config.forwardGetRequestForSrcsMap = lo.SliceToMap(config.GetStringSliceVar([]string{"adjust"}, "Gateway.webhook.forwardGetRequestForSrcs"), func(item string) (string, struct{}) {
		return item, struct{}{}
	})

    // usage
	if r.Method == "GET" {
		if _, ok := webhook.config.forwardGetRequestForSrcsMap[sourceDefName]; !ok {
			return
		}
	}
```

Reply:
Done